src/writing-policies/spec/06-signature-verifier-policies.md

```diff
@@ -0,0 +1,147 @@
+# Signature verifier policies
+
+Kubewarden implements support for the [Sigstore](https://www.sigstore.dev/)
+standard. This allows to implement a Secure Supply Chain for your cluster.
+
+Part of that is to ensure that all container images run the in the cluster are
+signed and verified, proving that they come from their stated authors, and
+haven't been tampered with (for other concerns, see the docs on [how we implement a Secure Supply Chain for the policies themselves](TODO link to https://github.com/kubewarden/docs/pull/100/files)).
+
+Sigstore signatures of containers are stored as layers of the container image.
+Policies that want to check Sigstore signatures of containers need then to check
+those container layers, and would need to pull the signature layers to see the
+signatures themselves.
```

Comment:
This feels confusing, at least to me. The way I read it, it looks like the signatures are stored inside of layers of the image that is being signed.

I would write this paragraph in a slightly different way:

```suggestion
Sigstore signatures are stored inside of container registries, next to the OCI object being signed;
be it a container image or a more generic OCI artifact,  like a Kubewarden policy.
Given an object to be signed, all its signatures are going to be stored as layers of a special OCI
object created by sigstore. 
Policies that want to check Sigstore signatures of containers need then to check
those layers, and would need to pull the signature layers to see the
signatures themselves.
```